tests/Organization.py

```diff
@@ -742,11 +742,7 @@ def testRemoveCustomProperty(self):
             self.org.get_custom_property("property_1")
 
     def testGetSelfHostedRunners(self):
-        # XXX: Organization BeaverSoftware does not exist
-        # https://api.github.com/orgs/BeaverSoftware
-        org = self.g.get_organization("ClickHouse")
-        runners = org.get_self_hosted_runners()
-        self.assertEqual(runners.totalCount, 602)
+        runners = self.org.get_self_hosted_runners()
```

Comment:
You reverted the tests for som reason. Could you, please, do `git reset --hard 00837740fd2d5b958fd7eaf56636a1e51e8c008b && git push --force-with-lease`?

It looks like something went wrong, because you overwrote the `setUp` step